hms-app/src/components/card-details/VotingButton.tsx

```diff
@@ -192,18 +192,14 @@ export function VoteButtons(props: IProps) {
   }
 
   return (
-    <Card.Section className={classes.noBorderSection}>
```

Comment:
pls remove card and stack from the imports :) 
also styles and classed dont seem to be used anymore